src/styles/_sizes.media.scss

```diff
@@ -5,6 +5,42 @@ $medium-width: 1220px;
 $preview-small-width: 849px;
 $preview-medium-width: 1000px;
 
+$size-small-max: 374px;
+$size-medium-min: 375px;
+$size-medium-max: 767px;
+$size-large-min: 768px;
+$size-large-max: 1023px;
+$size-xlarge-min: 1024px;
+
+$SIZE_SMALL: '(max-width: #{$size-small-max})';
```

Comment:
We typically use lower-hyphenated-case for all variables. Can we switch and then clarify the naming?